### ISSUE TITLE
[FEATURE] Nicer support for bookmarks

### ISSUE THREAD

**Issue by limelier** [adriankarlen/textfox#31]
Hey, 

Love the skin. Sideberry is also pretty cool, I'm glad I gave it a try.

Here's one thing that would be cool: slightly better support for bookmarks. I like to have my bookmark bar contents shown on my main bar at the top, but either way they could use some spacing or an outline or a background or something to make them stand out more against each other.

Current look:
![image](https://github.com/user-attachments/assets/a473daed-62d5-4fb6-8fb3-5349f84dad73)

I know Sideberry already has a thing for bookmarks, but I find it less nice to use than just having all of my folders listed out like this.

**Comment by adriankarlen** [comment]
I think you are missunderstanding the theme, I dont do anything in particular with bookmarks other than centering them within the bookmarks toolbar. Could it be that you've accidentally disabled [Bookmarks Toolbar](https://support.mozilla.org/en-US/kb/bookmarks-toolbar-display-favorite-websites)? If you set it to always show it should appear inside the navbar.

**Comment by limelier** [comment]
I guess the items being so close together might just be the fact that the folders have no icons or some quirk of the font. A little extra spacing between items or some other styling could be nice, still, but I suppose I can also add this myself if you have no problems with the items being too close together.

**Comment by adriankarlen** [comment]
I don't display my bookmarks the way you do, but I am also unable to reproduce the way it looks for you, what are you putting in the toolbar customization? Since you are having bookmarks in the navbar.

I have made som modifications to the theme to allow for a separated Bookmarks Toolbar, this is how it looks for me when I enable it.
![image](https://github.com/user-attachments/assets/471e7a43-5336-4260-88ec-0e01bbe87ef5)

**Comment by limelier** [comment]
You can drag and drop the bookmark toolbar contents into the main bar from the customize menu. I use it for its compactness.

-------- Original Message --------

**Comment by limelier** [comment]
It's also possible that I am missing some of the latest changes, if you've done anything to the bookmarks since I made the thread. I'll download the latest version and reply with a new screenshot when I can.

-------- Original Message --------

**Comment by adriankarlen** [comment]
Yeah, the changes were made after you submitted this issue. Okay, I understand, I've never used that method before personally. I'll see if I can come up with something that makes it easier to differntiate between bookmarks.

**Comment by limelier** [comment]
![image](https://github.com/user-attachments/assets/436662ac-a43b-48cb-bf82-dca37dc7ca8a)
I've updated to the latest version. The bookmark text has changed to gray, but otherwise they look pretty similar.

Thanks a lot for looking into it! I think the main reason I have a problem with the spacing and you don't is because your bookmark folders don't use any spaces in their names, while mine do.

**Comment by adriankarlen** [comment]
Hi, PR #44 adds a bit of increased padding for the bookmark items. Hopefully this helps your usecase. All the best.
